python/turbine_models/custom_models/sd_inference/vae_runner.py

```diff
@@ -0,0 +1,119 @@
+import argparse
+from turbine_models.model_runner import vmfb_runner
+from transformers import CLIPTokenizer
+from iree import runtime as ireert
+import torch
+
+parser = argparse.ArgumentParser()
+
+parser.add_argument(
+    "--vmfb_path", type=str, default="", help="path to vmfb containing compiled module"
+)
+parser.add_argument(
+    "--external_weight_path",
+    type=str,
+    default="",
+    help="path to external weight parameters if model compiled without them",
+)
+parser.add_argument(
+    "--compare_vs_torch",
+    action="store_true",
+    help="Runs both turbine vmfb and a torch model to compare results",
+)
+parser.add_argument(
+    "--hf_model_name",
+    type=str,
+    help="HF model name",
+    default="CompVis/stable-diffusion-v1-4",
+)
+parser.add_argument(
+    "--hf_auth_token",
+    type=str,
+    help="The Hugging face auth token, required for some models",
+)
+parser.add_argument(
+    "--device",
+    type=str,
+    default="local-task",
+    help="local-sync, local-task, cuda, vulkan, rocm",
+)
```

Comment:
Instead of copy pasting these we can have a single generic_flags.py and inherit parser from it, adding unique flags in these individual runners.

Reply:
Yeah, was thinking of that but looks like flags are kind of in flux with things being added like batch, height, and width to the sd models recently and wanted to let it settle a bit. Will add a todo to address